world-client/src/state/state.js

```diff
@@ -1,12 +1,83 @@
+import * as SimplexNoise from "simplex-noise";
+import * as random from "random";
+import * as seedrandom from "seedrandom";
+
 export default class State {
 
   constructor(wop) {
     this.wop = wop;
+    this.terrainFrequency = 0.1;
     this.state = {
       players: [],
+      gameSeed: "211231334",
+      worldTiles: [],
+      tileSize: 0.25
     };
   }
 
+  generateWorld(wop) {
+    // gre skozi vsa polja
+    for (let i = 0; i < 65; i++) {
+      this.state.worldTiles[i] = [];
+    }
+    // GRASS and MUD
+    for (let i = 0; i < 65; i++) {
+      for (let j = 0; j < 65; j++) {
+        this.state.worldTiles[i][j] = { tileType: 0 };
+        var item = this.state.worldTiles[i][j].tileType;
+        // algoritem
+        var simplex = new SimplexNoise(this.state.gameSeed), value2d = simplex.noise2D(i * this.terrainFrequency, j * this.terrainFrequency);
+        value2d += 1;
+        value2d = value2d / 2;
+        if (value2d < 0.7) {
+
+          this.state.worldTiles[i][j].tileType = TILE_TYPES.GRASS_BIOME;
+        } else {
+          this.state.worldTiles[i][j].tileType = TILE_TYPES.MUD_BIOME;
+        }
+      }
+    }
+    // WATER and OBSTACLE
+    for (let i = 0; i < 65; i++) {
+      for (let j = 0; j < 65; j++) {
+        var item = this.state.worldTiles[i][j].tileType;
+        // algoritem
+        var simplex = new SimplexNoise(this.state.gameSeed), value2d = simplex.noise2D(i * this.terrainFrequency, j * this.terrainFrequency);
+        value2d += 1;
+        value2d = value2d / 2;
+
+        if (value2d < 0.1) {
```

Comment:
Config...
Zgolj razmišlanje. Če se da to kot config, se lahko naredi različne presete in se tega pošlje kot parameter. Npr. bush/forest se močno dvigne in postane "jungle" mapa. Sand = 0.9 (če bo) puščava ipd...